backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -458,11 +458,11 @@ class TestOperator extends WholeStageTransformerSuite {
       .select($"DecimalCol".cast(DecimalType(38, 33)))
       .select(col("DecimalCol"))
       .agg(avg($"DecimalCol"))
-    assert(result.collect()(0).get(0).toString.equals("0.0345678900000000000000000000000000000"))
+    assert((result.collect()(0).get(0).toString.toDouble - d).abs < 0.00000000001)
```

Comment:
@rui-mo It is also due to https://github.com/facebookincubator/velox/pull/6051#issuecomment-1731028215? If yes, can we also add the comment?

Reply:
Updated.